src/Microsoft.Health.Dicom.Functions/Update/Models/CleanupBlobArguments.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.Model;
+using Microsoft.Health.Dicom.Core.Features.Partitioning;
+
+namespace Microsoft.Health.Dicom.Functions.Update.Models;
+
+public sealed class CleanupBlobArguments
```

Comment:
Again, changing the input to an activity or orchestration is a breaking change if it's in use. If it's in use, rename the function or make the input backwards compatible.

Reply:
Zeiss can test this feature any moment